backend/config/config.go

```diff
@@ -394,3 +399,26 @@ func (config *Config) migrateInfura(appConfig AppConfig) AppConfig {
 	migrate(&appConfig.Backend.RETH, "etherscan+https://api-rinkeby.etherscan.io/api")
 	return appConfig
 }
+
+// migrateFiatList moves fiatList from appconf.Frontend to appconf.Backend.
+// This is because with the account portfolio feature, backend needs to know
+// which fiat currencies are enabled to fetch historical exchange rates.
+func migrateFiatList(appconf *AppConfig) {
+	if len(appconf.Backend.FiatList) > 0 {
+		return // already migrated
+	}
+	frontconf, ok := appconf.Frontend.(map[string]interface{})
+	if !ok {
+		return // can't migrate
+	}
+	if fiats, ok := frontconf["fiatList"].([]string); ok {
```

Comment:
I think this conversion does not work.

if you do it without the `ok` check, the panic says:

```
panic: interface conversion: interface {} is []interface {}, not []string [recovered]
	panic: interface conversion: interface {} is []interface {}, not []string
```

Reply:
it's possible your ~/.config/bitbox/config.json has a different value? could you check please.

This is what you get for "opaque" things when trying to parse them instead of endorsing strong types. :)

Reply:
You can see how frontend was storing the list:

![image](https://user-images.githubusercontent.com/25405/95581225-ed437700-0a38-11eb-8762-903601816c97.png)

That's why I picked `[]string` as the type.

Reply:
I know what's happening here. Check out the rates.tsx in frontend:

![image](https://user-images.githubusercontent.com/25405/95587750-c5f1a780-0a42-11eb-9a2a-01659792f709.png)

If you never changed fiat settings, you'll never have it stored in the app config.json and it's why this migration fails. I'll fix it.

Reply:
Although ILS is enabled in your screenshot., which is not in the default "selected" list... I wonder how that happened.

Reply:
> Although ILS is enabled in your screenshot., which is not in the default "selected" list... I wonder how that happened.

It was not the default selection, it was my selection...

Reply:
So, there were two problems. 1st is like you said, []interface{} vs []string. 2nd is the frontend never stored fiatList or fiatCode in the config.json until you actually changed it from the UI.

Anyway, all should work now.